### PR TITLE
Added nginx.conf.erb file to set up config for osumbchallenges.dev

### PR DESCRIPTION
**Pull request by atareshawty**
Nginx will give us the ability to hit osumbchallenges.dev rather than localhost:3000. The main reason for this is to be able to have a redirect URL to give to OSU's SSO for local testing (osumbchallenges.dev/login/callback or something like that)

### CODE REVIEW

File: Brewfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 brew 'postgresql'
 brew 'node'
+brew "nginx"
```
Comment (tarebyte) [inline Brewfile]: @atareshawty you need to add `brew tap github/bootstrap` to the top of this file see: https://github.com/github/homebrew-bootstrap#usage

File: config/NGINX.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# If you want to use Nginx for a url without a port
```
Comment (tarebyte) [inline config/NGINX.md]: I'm sorry I remember why this works for me out of the box.

I have my machine setup with https://github.com/mikemcquaid/strap which sets this up for you.

**Comment by tarebyte** [comment]
Outside of my two comments this looks good @atareshawty

File: Brewfile (hunk 1 above)
Comment (atareshawty) [inline Brewfile]: Why? It works without

Comment (tarebyte) [inline Brewfile]: Because other if someone installs this fresh the nginx part of th bootstrap won't work.

**Pull request merged**